server/README.md

```diff
@@ -29,3 +29,14 @@ See [example.env](example.env) for information on Auth0 environment variables re
   ```sh
   yarn watch
   ```
+### Deploying to GCloud
+
+This `/server` folder is designed to be deployed to Google Cloud App Engine instance. It
+can be deployed from a local machine after configured.
+
+Using the `gcloud` CLI, you can deploy using
+```
+npm install
+npm run build
+gcloud app deploy
```

Comment:
Before running npm install, should we make sure the node version is correct?
Before running `gcloud app deploy`, I think we need to make sure the correct gcloud app is selected. We should add `gcloud init` step before gcloud app deploy.